markdown_stoplight/changelog.md

```diff
@@ -1,120 +1,147 @@
 ### Change Log
 
-This change log covers significant updates and changes to the Beyond
-REST API.
+This change log covers significant updates and changes to the Beyond REST API.
+
+**2019-01-15**
+
+**New**
+
+Allow user creation without specifying a password.
+
+Add new endpoint [Create variation product](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/products-create-with-variations).
```

Comment:
I wonder if we already want to document this, while not being able to ship this feature to all merchants, yet?

Reply:
I talked to Jan and Timo, and it's fine for them.